libpod/container_config.go

```diff
@@ -286,10 +286,13 @@ type ContainerNetworkConfig struct {
 	// DNS options to be set in container resolv.conf
 	// With override options in host resolv if set
 	DNSOption []string `json:"dnsOption,omitempty"`
+	// UseImageHostname indicates that /etc/hostname should not be
+	// bind-mounted inside the container.
+	UseImageHostname bool `json:"useImageHostname"`
 	// UseImageHosts indicates that /etc/hosts should not be
 	// bind-mounted inside the container.
 	// Conflicts with HostAdd.
-	UseImageHosts bool
+	UseImageHosts bool `json:"useImageHosts"`
```

Comment:
Isn't this a breaking change? The old container config contains `UseImageHosts` while the new json uses `useImageHosts` as such the unmarshal of an old config no longer reads the key thus changing the behavior of an previously created container.